assets/views/home/header.ejs

```diff
@@ -33,6 +33,7 @@
             <li><a href="#" data-toggle="modal" data-target="#flairAppList">Flair App List</a></li>
             <li><a href="#" data-toggle="modal" data-target="#flairMod">Edit Flairs</a></li>
             <li><a href="/mod/banuser">Ban a User</a></li>
+            <li><a target="_self" href="/mod/downloadModmail" download="modmails.txt">Download Modmails</a></li>
```

Comment:
What do you want the modmail for? To search something, yes?

If you download the entire file, what would be the purpose of that? It is less steps, because the user isn't downloading something if they are just searching. It's ... unnecessary. I just don't see why we would need the file at all now.